r2ai/interpreter.py

```diff
@@ -756,6 +760,71 @@ def compress_messages(self, messages):
     # if msglen > 4096:
     #   ¡print("Query is too large.. you should consider triming old messages")
     return messages
+  
+  def process_tool_calls(self, tool_calls):
+    self.messages.append({ "tool_calls": tool_calls, "role": "assistant" })
+    for tool_call in tool_calls:
+      res = ''
+      if tool_call["function"]["name"] == "r2cmd":
+        args = json.loads(tool_call["function"]["arguments"])
+        sys.stdout.write('\x1b[1;32mRunning \x1b[4m' + args["command"] + '\x1b[0m\n')
+        # r2lang.cmd('e scr.color=0')
+
+        res = r2lang.cmd(args["command"])
```

Comment:
you can also disable colors temporaily with : args["command"] + "@e:scr.color=0"

Reply:
the thing is that want to capture the output of the command uncolored but display it colored. I just strip the ANSI before sending to LLM

Reply:
sounds good then we can probably remove the commented line above